src/inline-svg.directive.ts

```diff
@@ -141,7 +141,7 @@ export class InlineSVGDirective implements OnInit, OnChanges, OnDestroy {
   private _insertEl(el: Element) {
     if (this.replaceContents && !this.prepend) {
       if (this._prevSVG) {
-        this._prevSVG.remove();
+        this._el.nativeElement.removeChild(this._prevSVG);
```

Comment:
Well, it would actually have to be `.parentNode`. Damn IE. http://stackoverflow.com/questions/8685739/difference-between-dom-parentnode-and-parentelement